packages/web/src/components/gcds-link/stories/overview.mdx

```diff
@@ -0,0 +1,77 @@
+import { Meta, Canvas, Story } from '@storybook/blocks';
+import * as Link from './gcds-link.stories';
+
+<Meta of={Link} name="Overview" />
+
+# Link<br/>`<gcds-link>`
+
+_Also called: anchor, hyperlink, call to action._
+
+A navigational element that allows users to navigate to a new page, website or section on the current page.
+
+<Canvas
+  of={Link.Default}
+  story={{ inline: true }}
+/>
+
+## Examples
+<br/>
+
+### External
+<Canvas
+  of={Link.External}
+  story={{ inline: true }}
+/>
+
+### Download
+<Canvas
+  of={Link.Download}
+  story={{ inline: true }}
+/>
+
+### Email addresses
+<Canvas
+  of={Link.Email}
+  story={{ inline: true }}
+/>
+
+### Phone numbers
+<Canvas
+  of={Link.Phone}
+  story={{ inline: true }}
+/>
+
+### Sizes
+
+#### Size inherit
+<Canvas
+  of={Link.SizesInherit}
+  story={{ inline: true }}
+/>
+
+#### Size regular
+<Canvas
+  of={Link.SizesRegular}
+  story={{ inline: true }}
+/>
+
+#### Size small
+<Canvas
+  of={Link.SizesSmall}
+  story={{ inline: true }}
+/>
+
+
+## Resources
+
+<ul>
+  <li>
+    <gcds-link href="https://design-system.alpha.canada.ca/en/components/link/" target="_blank">Guidance</gcds-link>
```

Comment:
Noticed another small thing: can we set all the resources links to `size="regular"` please?

```suggestion
    <gcds-link size="regular" href="https://design-system.alpha.canada.ca/en/components/link/" target="_blank">Guidance</gcds-link>
```